docs/release-notes/next-fortuna/PT-165593709_pinned_env_in_noise

```diff
@@ -0,0 +1,2 @@
+* Introduces a pinned environment in State Channels noise protocol. This is
+  not backwards compatible
```

Comment:
If I understand this correctly:
* The noise-based state channel protocol has no versioning. ([Past draft](https://github.com/aeternity/aeternity/pull/2204).)
* New node and old node manage to connect at TCP and noise level, though when exchanging any meaningful message either one of them (depending on who requested whom what first) close the TCP connection.

Right?

Reply:
Yes